tests/keras/backend/backend_test.py

```diff
@@ -1702,6 +1702,107 @@ def test_sparse_concat(self):
             assert k_s_d.shape == k_d.shape
             assert_allclose(k_s_d, k_d, atol=1e-05)
 
+    @pytest.mark.skipif((K.backend() != 'mxnet'),
+                        reason='Testing only for MXNet backend')
+    def test_sparse_sum(self):
```

Comment:
I think it will be good idea to move all sparse tests to a separate file. may be - tests/keras/backend/sparse_test.py ?
It will be easier later for merge/rebase etc. and also cleanly separates the tests, that can be eventually contributed back to Keras tests

Reply:
sounds good - will do